Task1/Node/hash-generation.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+var wstream = fs.createWriteStream('readme-main.md');
+
+function genhash(filename) {
+    const crypto = require('crypto');
```

Comment:
though it is allowed to `require` a module anywhere, its always good habit to keep them at the beginning of the file.

Reply:
modified the code by writing require modules in the beginning of the code.